src/main/java/wolox/training/models/Users.java

```diff
@@ -24,12 +24,7 @@
     @Column(nullable = false)
     private LocalDate birthday;
 
-    @ManyToMany(targetEntity = Book.class, cascade = CascadeType.MERGE, fetch = FetchType.EAGER)
-    @JoinTable(
-            name = "user_books",
-            joinColumns = {@JoinColumn(name = "USERS_ID")},
-            inverseJoinColumns = {@JoinColumn(name = "BOOK_ID")}
-    )
+    @ManyToMany(cascade = {CascadeType.MERGE, CascadeType.REFRESH})
```

Comment:
Está bien haber sacado la relación de la tabla? 🤔 cómo identifica de dónde sacar la data?

Reply:
No pasa nad, ya que abajo en el tipo le indicamos que es de Book.

Reply:
Like Oriana says you can avoid de definition of the JoinTable and the magic of springboot and hibernate will know how to map this into the db. This is the diagram of the relationship  between user and book 
![image](https://user-images.githubusercontent.com/43348034/88113522-4d3c2700-cb88-11ea-93c8-7f30d790b428.png)

Reply:
omg , cool